### PR TITLE
chore: Make code "Remix" friendly

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Make code "remix" friendly.

- [x] - Remove specific handlers for canvas and builder
- [x] - Use naive remix root.tsx entri.client and entri.server (we do not use hydrate on client, we inject stitches on the server, anything else is the same as in default remix project)
- [x] - Use `_ui` layout for builder, dashboard and login pages
- [x] - Render canvas layout in $.tsx
- [x] - Remove root.tsx 

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/entry.server.tsx [hunk 1]
```diff
@@ -1,45 +1,115 @@
...
+    request,
+    responseStatusCode,
+    responseHeaders,
+    remixContext
   );
+}
 
-  responseHeaders.set("Content-Type", "text/html");
+// eslint-disable-next-line func-style
+async function* injectStyle(stream: PassThrough, styleContent: string) {
```
Comment (istarkov) [inline apps/builder/app/entry.server.tsx:37]: This part is ours (inject stitches) everything else is naive Remix entri.server

File: apps/builder/app/entry.client.tsx [hunk 2]
```diff
@@ -1,18 +1,8 @@
 import { RemixBrowser } from "@remix-run/react";
 import { startTransition, StrictMode } from "react";
-import { createRoot, hydrateRoot } from "react-dom/client";
+import { createRoot } from "react-dom/client";
 
 startTransition(() => {
```
Comment (istarkov) [inline apps/builder/app/entry.client.tsx:5]: We use client rendering, there is no need in hydrate

File: apps/builder/app/entry.server.tsx [hunk 3]
```diff
@@ -1,45 +1,115 @@
...
+  const styleTag = `<style>${styleContent}</style>`;
+  let injected = false;
```
Comment (kof) [inline apps/builder/app/entry.server.tsx:39]:
```suggestion
  let isInjected = false;
```

File: apps/builder/app/entry.server.tsx [hunk 4]
```diff
@@ -1,45 +1,115 @@
...
+  for await (const chunk of stream) {
+    if (injected) {
+      yield chunk;
+      continue;
+    }
 
-export default function handleRequest(
+    buffer += chunk.toString();
+
+    if (!injected && buffer.includes("</head>")) {
```
Comment (kof) [inline apps/builder/app/entry.server.tsx:50]:
```suggestion
    if (injected  === false && buffer.includes("</head>")) {
```

**Review by kof — APPROVED** [review]

**Pull request merged**